modules/home/assets/js/components/home-screen.js

```diff
@@ -1,14 +1,16 @@
 import { Container, Grid } from '@elementor/ui';
 
 import TopSection from './top-section';
-import BlockSection from './block-section';
 
 const HomeScreen = () => {
+	const videoUrl = 'https://elementor.com/academy/getting-started-with-elementor/',
+		ctaUrl = elementorAppConfig.pages_url,
+		embedUrl = 'https://www.youtube.com/embed/icTcREd1tAg?si=40E8D1hdnu26-TXM';
+
 	return (
-		<Container sx={ { py: 4 } }>
-			<Grid sx={ { display: 'grid', gap: 3 } }>
-				<TopSection />
-				<BlockSection />
+		<Container sx={ { py: { xs: 2, md: 6 }, maxWidth: { md: '990px' } } }>
```

Comment:
Rotem will discuss the maxWidth with Hadar. He would like us to stick to the default values.